packages/groqd-playground/src/components/Playground.tsx

```diff
@@ -439,7 +442,17 @@ export default function GroqdPlayground({ tool }: GroqdPlaygroundProps) {
               <Stack space={3}>
                 <Box>
                   <Box paddingX={3} marginBottom={1}>
-                    <Label muted>Query</Label>
+                    <Label muted>
+                      Query{"  "}
+                      {query.query && (
+                        <span
```

Comment:
Should we give this a role of a button or use a button?

Reply:
ah yeah, probably. This was quick and dirty, but does deserve some aria context. I'll sneak that in here.